src/validators/relayer.py

```diff
@@ -30,16 +32,25 @@ async def get_validators(self, start_index: int, count: int) -> RelayerValidator
             resp = await session.post(f'{settings.relayer_endpoint}/validators', json=jsn)
             resp.raise_for_status()
             resp_json = await resp.json()
-            validators = [
-                RelayerValidator(
-                    public_key=v['public_key'],
-                    amount_gwei=v['amount_gwei'],
-                    signature=v['deposit_signature'],
-                    exit_signature=BLSSignature(Web3.to_bytes(hexstr=v['exit_signature'])),
+            validators = []
+            for validator_item in resp_json['validators']:
+                validator = Validator(
+                    public_key=validator_item['public_key'],
+                    amount_gwei=validator_item['amount_gwei'],
+                    signature=validator_item['deposit_signature'],
+                    exit_signature=BLSSignature(
+                        Web3.to_bytes(hexstr=validator_item['exit_signature'])
+                    ),
+                )
+                validators.append(validator)
+
+            multi_proof: MultiProof[tuple[bytes, int]] | None = None
+            if resp_json['proof'] is not None:
+                tx_validators = encode_tx_validator_list(validators)
+                leaves = list(zip(tx_validators, resp_json['proof_indexes']))
+                multi_proof = MultiProof(
+                    leaves=leaves, proof=resp_json['proof'], proof_flags=resp_json['proof_flags']
                 )
-                for v in resp_json['validators']
-            ]
             return RelayerValidatorsResponse(
-                validators=validators,
-                validators_manager_signature=resp_json['validators_manager_signature'],
+                validators=validators, validators_manager_signature=None, multi_proof=multi_proof
```

Comment:
Why validators manager signature is always None?

Reply:
reworked